update of a non installed package throws null pointer exception

update of a non installed package throws null pointer exception
```sh
vpc@linux-rogue:/data/git/nuts> ./nuts-dev -w="" update netbeans-launcher
execution failed of : update netbeans-launcher ; error was : java.lang.NullPointerException    
execution failed of : update netbeans-launcher ; error was : java.lang.NullPointerException
        at net.thevpc.nuts.runtime.standalone.wscommands.exec.DefaultNutsExecCommand.run(DefaultNutsExecCommand.java:161)
        at net.thevpc.nuts.boot.NutsBootWorkspace.runWorkspace(NutsBootWorkspace.java:1177)
        at net.thevpc.nuts.Nuts.runWorkspace(Nuts.java:233)
        at net.thevpc.nuts.Nuts.main(Nuts.java:84)
Caused by: java.lang.NullPointerException
        at net.thevpc.nuts.runtime.standalone.wscommands.update.DefaultNutsUpdateCommand.checkUpdates(DefaultNutsUpdateCommand.java:158)
        at net.thevpc.nuts.runtime.standalone.wscommands.update.DefaultNutsUpdateCommand.getResult(DefaultNutsUpdateCommand.java:59)
        at net.thevpc.nuts.runtime.standalone.wscommands.update.DefaultNutsUpdateCommand.update(DefaultNutsUpdateCommand.java:88)
        at net.thevpc.nuts.runtime.standalone.wscommands.update.DefaultNutsUpdateInternalExecutable.execute(DefaultNutsUpdateInternalExecutable.java:28)
        at net.thevpc.nuts.runtime.standalone.wscommands.exec.DefaultNutsExecCommand.run(DefaultNutsExecCommand.java:137)
        ... 3 more
```

--- a/core/nuts-runtime/src/main/java/net/thevpc/nuts/runtime/standalone/wscommands/update/DefaultNutsUpdateCommand.java
+++ b/core/nuts-runtime/src/main/java/net/thevpc/nuts/runtime/standalone/wscommands/update/DefaultNutsUpdateCommand.java
@@ -377,8 +377,7 @@ public class DefaultNutsUpdateCommand extends AbstractNutsUpdateCommand {
 //            }
 //        }
         if (d0 == null) {
-            return r;
-            //throw new NutsIllegalArgumentException(getSession(), NutsMessage.cstyle("%s is not yet installed for it to be updated.",id));
+            throw new NutsIllegalArgumentException(getSession(), NutsMessage.cstyle("%s is not yet installed for it to be updated.", id));
         }
         if (!d0.getInstallInformation().isDefaultVersion()) {
             shouldUpdateDefault = true;

